lib/install/management/appliance.go

```diff
@@ -67,14 +68,42 @@ func (d *Dispatcher) isVCH(vm *vm.VirtualMachine) (bool, error) {
 	return false, nil
 }
 
-func (d *Dispatcher) checkExistence(conf *metadata.VirtualContainerHostConfigSpec) error {
+func (d *Dispatcher) checkExistence(conf *metadata.VirtualContainerHostConfigSpec, settings *data.InstallerData) error {
 	defer trace.End(trace.Begin(""))
 
-	vm, err := d.findAppliance(conf)
+	var err error
+	d.vchPoolPath = fmt.Sprintf("%s/%s", settings.ResourcePoolPath, conf.Name)
```

Comment:
Using `path.Join()` is a bit more concise for constructing inventory paths, no need for a format string.